Documentation/Signoffs/CommunicationsSignoff.md

```diff
@@ -0,0 +1,64 @@
+# Communication Signoff
```

Comment:
function - it is unclear where and for what purpose the data is encrypted, also where would it be decrypted

buildables - need circuit diagram showing wiring connections - if it is a module that plugs directly onto another board that needs to be mentioned in lieu of a wiring diagram

buildables - the flowchart is unclear in a few places where it is waiting for something - doesn't the "wait" box imply waiting until something happens? if so, why is there a yes/no right after it? also is there a timeout or recovery mechanism to prevent it from waiting forever?

bom notes - also state do not initially purchase the cellular interface card

Reply:
Function - clarified that formatting and encrypting happen together.  I have also gave a very brief reason for encryption.

Buildables - I have created two circuit diagrams showing the UART connections and the names of their pins.  The power supply differences are also present.  All unused pins are omitted here for clarity.  

Buildables - the "Yes/No" after "Wait For Instructions" has been removed, no longer implying that the software is checking for something.  As for the timeout or recovery mechanism, there are watchdogs in place once instructions have been received and then onwards.  There are none for the "Wait For Instructions".  If the software is getting stuck at this stage, then something bigger is wrong because it should just sit there until told what to do.

Bom Notes - Additions made.